### PR TITLE
feat(VMenu): add contentProps prop

### PR DESCRIPTION
**Pull request by ricardovanlaarhoven**
## Description
Added contentAttributes prop to the VMenu so you can add any attribute to the vmenu's content

## Motivation and Context
When testing using cypress this is needed for selecting something in a VSelect. But i guess there could be numerous options.

## How Has This Been Tested?
Just use the playground and check if there are attributes in the content

## Markup:

<details>

```vue
<template>
  <v-app id="inspire">
    <v-container >
      <v-select
          :items="[{text:'a', value:1}, {text: 'b', value: 2}]"
          :menu-props="{contentClass:'test123', contentAttributes:{test:123}}"></v-select>
    </v-container>
  </v-app>
</template>

<script>
  export default {

  }
</script>

```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [x] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

**Comment by ricardovanlaarhoven** [comment]
Hi Maintainers,

I'm participating in hacktoberfest again. A initiative to get more people to contribute to open source.
Hacktoberfest is now opt-in for maintainers, so only contribute to projects that indicate they’re looking for Hacktoberfest PR/MRs.

YOUR PR/MRS MUST BE IN A REPO TAGGED WITH THE “HACKTOBERFEST” TOPIC, OR HAVE THE “HACKTOBERFEST-ACCEPTED”


Could you add the “HACKTOBERFEST-ACCEPTED” label to this PR when you think this PR is valid?

**Comment by KaelWD** [comment]
This exists in v3 but is called `contentProps`, and also applies to other detached components like VDialog, VTooltip, and VSnackbar

**Comment by ricardovanlaarhoven** [comment]
> This exists in v3 but is called `contentProps`, and also applies to other detached components like VDialog, VTooltip, and VSnackbar

Is it ok to add this to v2? or would that never be accepted?

**Comment by KaelWD** [comment]
Yeah best to keep the same name when backporting stuff though.

File: packages/vuetify/src/components/VMenu/VMenu.ts [hunk 1]
```diff
@@ -73,6 +75,10 @@ export default baseMixins.extend({
     },
     offsetX: Boolean,
     offsetY: Boolean,
+    openOnClick: {
+      type: Boolean,
+      default: true,
+    },
```
Comment (KaelWD) [inline packages/vuetify/src/components/VMenu/VMenu.ts:81]: How is this relevant?

**Comment by ricardovanlaarhoven** [comment]
Unfortunately i don't have time to look into your questions now. I don't have the anwsers since it's a while ago i made those changes.
What i know is that I looked into other components with the same functionality. I think components that also use the menuAble mixin.

I'll have time after the hollidays.

**Review by johnleider — APPROVED** [review]
